### ISSUE TITLE
EKS Cluster documentation unclear about logging and endpoint access configuration

### ISSUE THREAD

**Issue by ConaghFM** [awsdocs/aws-cloudformation-user-guide#840]
The [CloudFormation docs for EKS Clusters](https://github.com/awsdocs/aws-cloudformation-user-guide/blob/main/doc_source/aws-resource-eks-cluster.md) refer to "`endpointPublicAccess` and `endpointPrivateAccess` parameters" and "the `logging` parameter". It appears these sections have been copied and pasted from the corresponding [API reference page](https://docs.aws.amazon.com/eks/latest/APIReference/API_CreateCluster.html).
There don't seem to be any corresponding properties listed in the CloudFormation docs for configuring endpoint access and logging. Is there a way to do that in CloudFormation?

**Comment by MitchellJeppson** [comment]
Agreed, the documentation is confusing. This AWS example uses endpointPublicAccess and endpointPrivateAccess: [eks-quickstart](https://github.com/aws-quickstart/quickstart-amazon-eks/blob/main/templates/amazon-eks-controlplane.template.yaml)

But creating a similar stacks results in: 
`Property validation failure: [Encountered unsupported properties in {/ResourcesVpcConfig}: [EndpointPublicAccess, PublicAccessCidrs, EndpointPrivateAccess]]`
